rustls/src/server/tls13.rs

```diff
@@ -880,23 +880,37 @@ impl ExpectFinished {
         })
     }
 
-    fn emit_stateless_ticket(
+    fn emit_ticket(
         handshake: &mut HandshakeDetails,
         sess: &mut ServerSessionImpl,
         key_schedule: &KeyScheduleTraffic,
     ) {
         let nonce = rand::random_vec(32);
         let plain = get_server_session_value(handshake, key_schedule, sess, &nonce).get_encoding();
-        let ticket = match sess.config.ticketer.encrypt(&plain) {
-            Some(t) => t,
-            None => return,
+
```

Comment:
I think it would be a bit clearer if we just wrote:

```
let stateless = sess.config.ticketer.enabled();
if stateless {
...
```

Reply:
Fair point, fixed.